src/extension.ts

```diff
@@ -1137,6 +1137,14 @@ class ExtensionManager implements vscode.Disposable {
     await logging.showLogFile();
   }
 
+  activeFolder() : string  {
+    let thisActiveFolder: string = '';
+    if(this._folders.activeFolder){
+      thisActiveFolder = this._folders.activeFolder.folder.name;
+    }
+    return thisActiveFolder;
+  }
```

Comment:
You can do this which is equivalent:
```
activeFolder() : string { return this._folders.activeFolder?.folder.name || ''; }
```

Also, do you only need the name or the full path? Will there be problems when there are multiple folders with the same name?

Reply:
This is a point. For now both are implemented, the active folders name and path, respectively.